installation/install-by-dmg.md

```diff
@@ -14,7 +14,8 @@ For macOS, `td-agent` is distributed as `.dmg` installer.
 
 Download and install the `.dmg` package:
 
-* [Download](https://td-agent-package-browser.herokuapp.com/3/macosx)
+* [td-agent v4 (fluentd 1.11.x)](https://td-agent-package-browser.herokuapp.com/4/macosx)
```

Comment:
`(fluentd 1.11.x)` isn't needed because td-agent 4.1.1 is the first release of td-agent 4 for macOS and it uses fluentd 1.12.x, and 4.2.0 will use fluentd 1.13.x.

Reply:
I've fixed it.